.github/workflows/ios_hotfix.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
     outputs:
       release_branch_name: ${{ steps.make_release_branch.outputs.release_branch_name }}
-      asana_task_url: ${{ steps.create_release_task.outputs.asana_task_url }}
+      asana_task_url: ${{ steps.make_release_branch.outputs.asana_task_url }}
```

Comment:
Fortunately neither of the outputs here nor in the macOS hotfix workflow is in use, so we could remove all outputs from these two.